app/_how-tos/send-asychronous-llm-requests-to-save-costs.md

```diff
@@ -0,0 +1,289 @@
+        {"custom_id": "prod5", "method": "POST", "url": "/v1/chat/completions", "body": {"model": "gpt-4o-mini", "messages": [{"role": "system", "content": "You are a helpful assistant."}, {"role": "user", "content": "Write a concise product description for a compact and durable travel backpack with laptop compartment."}], "max_tokens": 60}}
+        EOF
+        ```
+  entities:
+    services:
+      - files-service
+      - batches-service
+    routes:
+      - files-route
+      - batches-route
+
+cleanup:
+  inline:
+    - title: Clean up Konnect environment
+      include_content: cleanup/platform/konnect
+      icon_url: /assets/icons/gateway.svg
+    - title: Destroy the {{site.base_gateway}} container
+      include_content: cleanup/products/gateway
+      icon_url: /assets/icons/gateway.svg
+---
+
+## Configure AI Proxy plugins
+
+Configure two separate AI Proxy plugins: one for the `llm/v1/files` route and another for the `llm/v1/batches` route. Each route type requires its own dedicated service and route to function correctly. In this setup, all requests to the files route are forwarded to `/files` endpoint, while batch requests go to `/batches` endpoint.
+
+
+AI Proxy plugin for the `route_type: llm/v1/files` :
+
+{% entity_examples %}
+entities:
+  plugins:
+    - name: ai-proxy
+      service: files-service
+      config:
+        model_name_header: false
+        route_type: llm/v1/files
+        auth:
+          header_name: Authorization
+          header_value: Bearer ${openai_api_key}
+        model:
+          provider: openai
+variables:
+  openai_api_key:
+    value: $OPENAI_API_KEY
+{% endentity_examples %}
+
+AI Proxy plugin for the `route_type: llm/v1/batches`:
+
+{% entity_examples %}
+entities:
+  plugins:
+    - name: ai-proxy
+      service: batches-service
+      config:
+        model_name_header: false
+        route_type: llm/v1/batches
+        auth:
+          header_name: Authorization
+          header_value: Bearer ${openai_api_key}
+        model:
+          provider: openai
+variables:
+  openai_api_key:
+    value: $OPENAI_API_KEY
+{% endentity_examples %}
+
+# Upload a .jsonl file for batching
+
+Use the following command to upload your [batching file](./#batch-jsonl-file) to the `/files` route:
+
+```bash
+curl localhost:8000/files -F purpose="batch" -F file="@batch.jsonl"
+```
+
+If the upload succeeds, you will see a JSON response like this:
+
+```json
+{
+  "object": "file",
+  "id": "file-abc123xyz456789lmn0pq",
+  "purpose": "batch",
+  "filename": "1.jsonl",
+  "bytes": 1672,
+  "created_at": 1751281528,
+  "expires_at": null,
+  "status": "processed",
+  "status_details": null
+}
+```
+
+{:.success}
+> **Note:** Copy the file ID from the response, you will need it to create a batch. Export it as an environment variable:
+>
+> ```bash
+>   export FILE_ID=YOUR_FILE_ID
+> ```
```

Comment:
Since this is a step I would not put it in a note (this applies to the next ones too)